website/settings/defaults.py

```diff
@@ -671,6 +671,11 @@ class CeleryConfig:
                     'batch_size_stuck': 10
                 }
             },
+            'monitor_registration_bulk_upload_jobs': {
+                'task': 'api.providers.tasks.monitor_registration_bulk_upload_jobs',
+                'schedule': crontab(minute=0, hour=7),  # Daily 03:00 a.m. EDT
```

Comment:
No, this is something we need to discuss and convey to the user. The frequency and time is temporary here.

I set once per day since to avoid the creation competing for resources. I set 3 a.m to avoid other tasks.

Reply:
As discussed, it now runs every 3 hours.